examples/go.mod

```diff
@@ -2,7 +2,10 @@ module github.com/deepmap/oapi-codegen/v2/examples
 
 go 1.20
 
-replace github.com/deepmap/oapi-codegen/v2 => ../
+replace (
+	github.com/deepmap/oapi-codegen/v2 => ../
+	github.com/oapi-codegen/runtime v1.1.0 => ../../../sonasingh46/runtime
```

Comment:
FYI this can be done locally using https://go.dev/doc/tutorial/workspaces so we don't require changes in the `go.mod`s

Reply:
Thanks. I will try that. I readyy never used go workspace